packages/lesswrong/components/comments/CommentsSortBySelector.tsx

```diff
@@ -0,0 +1,47 @@
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import React from 'react';
+import { useLocation } from '@/lib/routeUtil';
+import qs from 'qs'
+import { isEmpty } from 'underscore';
+import type { Option } from '../common/InlineSelect';
+import { useNavigate } from '@/lib/reactRouterWrapper';
+
+
+export const CommentsSortBySelector = ({setRestoreScrollPos}: {
```

Comment:
We use 2-space indentation; my guess is you have a 4-space default set in vscode